client/src/components/App/Content/InvestigationForm/InvestigationForm.tsx

```diff
@@ -1,8 +1,9 @@
 import React from 'react';
-import { Button } from '@material-ui/core';
 
-import useInvestigationForm from './useInvestigationForm';
+import PrimaryButton from 'commons/Buttons/PrimaryButton/PrimaryButton';
```

Comment:
Maybe because you use default import you can call it a button and not primary button

Reply:
I think that the name describes it better, since it is possible to have both primary and regular button under the same file